Code/GraphMol/MolDraw2D/catch_tests.cpp

```diff
@@ -43,7 +43,7 @@ namespace {
 // The hand-drawn pictures will fail this frequently due to the use
 // of random numbers to draw the lines.  As well as all the testHandDrawn
 // files, this includes testBrackets-5a.svg and testPositionVariation-1b.svg
-static const bool DELETE_WITH_GOOD_HASH = true;
+static const bool DELETE_WITH_GOOD_HASH = false;
```

Comment:
Maybe set this back to true, again?

Reply:
yep. I had that on my Todo list and just forgot